src/components/Participant/Participant.js

```diff
@@ -39,7 +39,7 @@ function Participant({
   const dispatch = useDispatch();
   const videoRef = React.createRef();
   const cssClassName = `Participant ${focus === 'user' ? 'focus' : ''}`;
```

Comment:
... because here it looks like `focus` is not the simple boolean the one-liner change would suggest

Reply:
Yes, you are right, it is wrong. Let me fix it.